src/main/java/com/atherys/towns/command/plot/BordersCommand.java

```diff
@@ -19,19 +19,13 @@
 @Aliases({"border", "borders"})
 @Description("Displays outline for town plots.")
 @Permission("atherystowns.plot.border")
-public class BordersCommand implements PlayerCommand, ParameterizedCommand {
-
-    @Override
-    public CommandElement[] getArguments() {
-        return new CommandElement[]{
-                GenericArguments.bool(Text.of("border"))
-        };
-    }
+public class BordersCommand implements PlayerCommand {
 
     @Nonnull
     @Override
     public CommandResult execute(@Nonnull Player source, @Nonnull CommandContext args) throws CommandException {
-        AtherysTowns.getInstance().getPlotBorderFacade().setPlayerViewBorderStatus(source, args.<Boolean>getOne("border").orElse(false));
+        AtherysTowns.getInstance().getPlotBorderFacade().setPlayerViewBorderStatus(
```

Comment:
Just remove the parameter from the method signature and grab the status in the facade's method.